modules/hostap/src/supp_events.c

```diff
@@ -437,3 +437,42 @@ int supplicant_generate_state_event(const char *ifname,
 
```

Comment:
In commit `hostap: add AP DPP wpa_msg event handler`, the commit message contains a C comment which seems unnecessary, could you remove the comment markers. The TODO text can stay of course.

Reply:
updated